setup.cfg

```diff
@@ -32,6 +32,7 @@ install_requires =
     tabulate
     tensorflow
     tensorflow-model-optimization<=0.7.5
+    torch
```

Comment:
Do we need this? This will force all installations of hls4ml to have both tf and torch, and since these have huge dependencies themselves it may bork user's environment with the partial updates. Ideally we would have hls4ml with optional extras installed to support various frontends (e.g., `pip install hls4ml[torch]`), but this may not be easy with the current codebase.

Also, if we go down this route, we should remove the checks for existence of pytorch on the system in `converters/__init__.py`

Reply:
I think we can get away with not making torch a requirement by moving the `from hls4ml.converters.pytorch_to_hls import parse_pytorch_model` from the `converters/__init__.py` directly into `config.py`. The only reason why I had put the import there is that it's there for keras. The question is, is there a reason to place these imports into the `converters/__init__.py`? If not, I can just move it and get rid of the requirement.